src/org/ohdsi/usagi/ui/Global.java

```diff
@@ -57,11 +45,16 @@
 	public static ApproveAllAction					approveAllAction;
 	public static ClearAllAction					clearAllAction;
 	public static ConceptInformationAction			conceptInfoAction;
+	public static AthenaAction						athenaAction;
+	public static GoogleSearchAction				googleSearchAction;
 	public static AboutAction						aboutAction;
 	public static ExportSourceToConceptMapAction	exportAction;
 	public static ExportForReviewAction				exportForReviewAction;
 	public static RebuildIndexAction				rebuildIndexAction;
 	public static ExitAction						exitAction;
 	public static String							vocabularyVersion;
+	public static Vector<String> 					conceptClassIds;
+	public static Vector<String> 					vocabularyIds;
+	public static Vector<String> 					domainIds;
```

Comment:
Please use `List<String>` in interfaces. If synchronised access is needed (multi-threaded access to these values), you can still set it to a `Vector` when you instantiate the variable. If multi-threaded access is not needed, this should be replaced by an `ArrayList`.